Source/Core/DolphinLib.vcxproj

```diff
@@ -70,31 +70,31 @@
   <ItemGroup>
     <Text Include="$(BuildInfoTemplate)" />
   </ItemGroup>
-  <UsingTask TaskName="GetProductVersion"
-    TaskFactory="CodeTaskFactory"
-    AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
+  <ItemGroup>
+    <ClCompile Include="Core\HW\DVD\AMMediaboard.cpp" />
+    <ClCompile Include="Core\HW\SI\SI_DeviceAMBaseboard.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="Core\HW\DVD\AMMediaboard.h" />
+    <ClInclude Include="Core\HW\SI\SI_DeviceAMBaseboard.h" />
+  </ItemGroup>
```

Comment:
We've been manually editing `DolphinLib.props` to add new source/header files instead of having visual studio add them to DolphinLib.vcxproj, because that lets us handle ARM- or x64-specific code better.